protostar/starknet/execution_environment.py

```diff
@@ -26,11 +26,11 @@ async def invoke(self, function_name: str) -> InvokeResultT:
         ...
 
     async def perform_invoke(
-        self, function_name: str
+        self, function_name: str, *args, **kwargs
```

Comment:
It is a nitpick but maybe nicer would be to explicitly name those test_args, test_kwargs and unpack them when calling 'return await func(*args, **kwargs).invoke()'? variadic args purpose may be a bit unclear when reading the code, WDYT?

Reply:
hmm.. This seems overly verbose to my taste. I kinda like `*args, **kwargs` approach because it mimics Python's style.